.github/workflows/global-ci-bundle.yml

```diff
@@ -0,0 +1,314 @@
+name: Global CI via Operator Bundle
+
+on:
+  workflow_call:
+    inputs:
+      artifact:
+        description: |
+          The name of the artifact storing custom images to be used during the CI run. All
+          images stored in the artifact will be automatically loaded.
+        type: string
+        required: false
+      bundle_image:
+        description: |
+          Image URI for the Konveyor operator bundle to be used when installing Konveyor.
+          This image MUST be publicly accessible (ie. not in the artifact).
+          For example, `ttl.sh/konveyor-operator-bundle-${SHA}:1h`
+        type: string
+        required: false
+        default: ""
+      namespace:
+        description: |
+          Namespace for the konveyor install.
+        type: string
+        required: false
+        default: ""
+      tackle_cr:
+        description: |
+          Full yaml encoded string representing the Tackle resource to be created.
+        type: string
+        required: false
+        default: ""
```

Comment:
This likely needs to be json encoded to make it easier to manage as input or from workflow dispatch (that can't handle multiline).

Reply:
I also wonder if there are certain things in the Tackle CR that MUST BE set in order to work correctly.